packages/runtime/src/runtime.ts

```diff
@@ -49,6 +50,7 @@ export class Runtime {
       args: []
     };
     this.appCounter = 0;
+    this.programMap = new Map<number, string[]>();
```

Comment:
Can we rename to `appPrograms` ? Or something similar (to represent application mapped to approval & clear programs)